cmd/nerdctl/internal_oci_hook.go

```diff
@@ -26,6 +26,12 @@ var internalOCIHookCommand = &cli.Command{
 	Name:   "oci-hook",
 	Usage:  "OCI hook",
 	Action: internalOCIHookAction,
+	Flags: []cli.Flag{
+		&cli.StringSliceFlag{
+			Name:  "add-host",
+			Usage: "Add a custom host-to-IP mapping (host:ip)",
+		},
+	},
```

Comment:
Why not use labels as in other flags?

Reply:
@AkihiroSuda you mean rename `add-host` to `labels` ?

Reply:
No, using labels in https://github.com/containerd/nerdctl/blob/master/pkg/labels/labels.go

Other ocihook features use these labels

Reply:
@AkihiroSuda  fixed